docs/source/release_notes/release-notes.rst

```diff
@@ -48,24 +74,12 @@ v0.14.0
            This will only affect deployments who run with more than one server (i.e. in HA mode).
            (`Pull Request <https://github.com/palantir/atlasdb/pull/873>`__)
 
-    *    - |breaking|
+    *    - |fixed|
```

Comment:
I don't know... according to the warn statements before,  it seemed like it was supported partially. We're really removing support for the partial configuration now so I would count it as breaking. @rhero thoughts?